### ISSUE TITLE
max_awaiting_rel config option is not honoured

### ISSUE THREAD

**Issue by biggms8x8** [emqx/emqx#10025]
### What happened?

Attempted to adjust the max_awaiting_rel option in emqx.conf but it is not picked up, the limit seems to only be taken from the default (100).  This shows on both the dashboard and when attempting to overwhelm the broker with QoS 2 messages.
The following options were added to emqx.conf (as shown in emqx-example.conf)

```
mqtt {
   max_awaiting_rel = infinity
}

zones.external.mqtt {
   max_awaiting_rel = infinity
}

zones.internal.mqtt {
   max_awaiting_rel = infinity
}
```
After a suggestion i also edited "data/configs/cluster-override.conf" to add the max_awaitng_rel option, this too is ignored

The setting isn't picked up if the listener is in a zone or default.

Additionally setting this in the dashboard in the MQTT->Session also has no effect.

### What did you expect to happen?

max_awaiting_rel to be read from emqx.conf
max_awaiting_rel to be honoured when set in the dashboard

### How can we reproduce it (as minimally and precisely as possible)?

Add the following to emqx.conf
```
mqtt {
   max_awaiting_rel = infinity
}

zones.external.mqtt {
   max_awaiting_rel = infinity
}

zones.internal.mqtt {
   max_awaiting_rel = infinity
}
```

Observe in the dashboard for a connection "QoS 2 Message Receive Queue" is still 100

On the dashboard change Configuration->MQTT->Session->Max Awaiting PUBREL to "infinity"
Observe in the dashboard for a connection "QoS 2 Message Receive Queue" is still 100

### EMQX version

5.0.16

**Comment by tigercl** [comment]
Hi, @biggms8x8 

If you have modified this configuration item through the Dashboard, then you need to modify `data/configs/cluster-override.conf` when you want to modify it again through the configuration file.

You can refer to [Configuration Files](https:// www.emqx.io/docs/en/v5.0/configuration/configuration.html)

**Comment by biggms8x8** [comment]
Thanks @tigercl .. Just tried modifying that, I've added that file to the original post.  So that's the `emqx.conf`, dashboard settings and `cluster-override.conf` that aren't honoured.

**Comment by tigercl** [comment]
@biggms8x8 Sorry, I check it again. I found that it does not work, this may be a bug.

We are investigating, I will synchronize the results with you ASAP.

**Comment by kedar-hatkar** [comment]
@zhongwencool @tigercl  @id  whether this issue is fixed in EMQX v5.0.20 ? i did not see any reference in release notes . 

If not then in which version, fix will be available ?

**Comment by zhongwencool** [comment]
Sorry for this delay. 
The fix PR already merge in to master, It will be ready in v5.0.21.
